shop_manager_app.rb

```diff
@@ -0,0 +1,128 @@
+require_relative 'lib/item_repository'
+require_relative 'lib/order_repository'
+require_relative 'lib/database_connection'
+
+class Application
+  def initialize(database_name, io, item_repository, order_repository)
+    DatabaseConnection.connect(database_name)
+    @io = io
+    @item_repository = item_repository
+    @order_repository = order_repository
+  end
+
+  def run
+    print_main
+    process_main
+  end
+
+  def print_main
+    @io.puts "Welcome to the Shop Manager!",""
+    @io.puts "What would you like to do?"
+    @io.puts " 1 - list all shop items"
+    @io.puts " 2 - create a new item"
+    @io.puts " 3 - list all orders"
+    @io.puts " 4 - create a new order",""
+    @io.puts "Enter you choice: "
+    @selection = @io.gets.chomp
+  end
+
+  def process_main
```

Comment:
I like that there is a separation of methods so each method has a single responsibility